Cookbook generation failure missing responsible file indicator

Currently cookbook generation does not provide any debug information. Therefore erroneous translation files are hard to track down.

```
Example:
-- Configuring done
-- Generating done
-- Build files have been written to: /home/vagrant/shogun/build
[4/5] Generating examples from meta-language
FAILED: cd /home/vagrant/shogun/build/examples/meta && /usr/bin/python /home/vagrant/shogun/examples/meta/generator/generate.py -i /home/vagrant/shogun/examples/meta/src -o /home/vagrant/shogun/build/examples/meta -t /home/vagrant/shogun/examples/meta/generator/targets -g /home/vagrant/shogun/build/src/shogun/tags --store-vars
Traceback (most recent call last):
  File "/home/vagrant/shogun/examples/meta/generator/generate.py", line 120, in <module>
    includedTargets=args.targets, storeVars=storeVars)
  File "/home/vagrant/shogun/examples/meta/generator/generate.py", line 44, in translateExamples
    targets.append(json.load(tFile))
  File "/usr/lib/python2.7/json/__init__.py", line 290, in load
    **kw)
  File "/usr/lib/python2.7/json/__init__.py", line 338, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python2.7/json/decoder.py", line 366, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python2.7/json/decoder.py", line 384, in raw_decode
    raise ValueError("No JSON object could be decoded")
ValueError: No JSON object could be decoded
ninja: build stopped: subcommand failed.
```

--- a/examples/meta/generator/generate.py
+++ b/examples/meta/generator/generate.py
@@ -37,10 +37,15 @@ def translateExamples(inputDir, outputDir, targetsDir, ctagsFile,
         # Ignore targets not in includedTargets
         if includedTargets and not os.path.basename(target).split(".")[0] in includedTargets:
             continue
-
-        with open(os.path.join(targetsDir, target)) as tFile:
-            targets.append(json.load(tFile))
-
+        
+        translate_file = os.path.join(targetsDir, target)
+        
+        with open(translate_file) as tFile:
+            try:
+                targets.append(json.load(tFile))
+            except Exception, err:
+                print("Error translating file: %s\n%s" % (translate_file, err))
+    
     # Translate each example
     for dirRelative, filename in subfilesRelative(inputDir, filter_by=lambda x: x.lower().endswith('.sg')):
         print("Translating {}".format(os.sep.join([dirRelative,filename])))

